packages/teleterm/src/mainProcess/windowsManager.ts

```diff
@@ -38,6 +38,7 @@ export class WindowsManager {
       backgroundColor: theme.colors.primary.darker,
       minWidth: 400,
       minHeight: 300,
+      autoHideMenuBar: true,
```

Comment:
This one is interesting though, I wonder if Windows users usually expect that bar to be present?

Reply:
I don't think so. If I remember correctly, starting from Windows 7, the menu bar was hidden by default and you had to press `Alt` to show it.

Another reason is that we don't show anything interesting in that menu ;)